tests/provision/facts/main.fmf

```diff
@@ -1,4 +1,10 @@
-summary: Check that guest facts are discovered and reported correctly
+/guest-facts:
```

Comment:
Let's deduplicate `facts` from the test name `/tests/provision/facts/guest-facts` --> `/tests/provision/facts/guest`.

Reply:
Done in 199351da